lighthouse-core/config/config.js

```diff
@@ -175,6 +177,31 @@ function validateCategories(categories, audits, auditResults, groups) {
         throw new Error(`${audit.id} references unknown group ${audit.group}`);
       }
     });
+
+    // Validate certification (if any).
+    if (category.certification) {
```

Comment:
> do we want to validate any certification in the config

like just check that the audits a certification is requesting will be run? We could do that, but I'm already running into issues with filtering the config with `onlyAudits`. For this version of validation, I was going to drop the `certification` property if `onlyAudits` filtered out a required audit rather than show an error. If we want to validate any certification in the config, I'm not sure what to do in that case...strip out the certification altogether?